draft-ietf-rats-architecture.md

```diff
@@ -907,7 +907,7 @@ in {{TCG-DICE-SIBDA}} or the symmetric mode described in
 
 Thus, trusting a Verifier might be expressed by having the Relying
 Party store the Verifier's key or certificate in its trust anchor store, or might
-be expressed by storing the public key or certificate of an entity (e.g., a Certificate Authority) that is
+be expressed by storing the public key or certificate of an entity (e.g., a Certification Authority) that is
```

Comment:
I Googled "CA" and found many uses of Certificate Authority. But I looked at RFC 4949 and found it recommends using Certification Authority.

~~~~
$ certificate authority
   (D) Synonym for "certification authority".

   Deprecated Term: IDOCs SHOULD NOT use this term; it suggests
   careless use of the term "certification authority", which is
   preferred in PKI standards (e.g., [X509, R3280]).
~~~~